XPT2046_Touchscreen.cpp

```diff
@@ -46,7 +46,7 @@ bool XPT2046_Touchscreen::begin()
 #ifdef ESP32
 void IRAM_ATTR isrPin( void )
 #else
-void isrPin( void )
+void ICACHE_RAM_ATTR isrPin( void )
```

Comment:
Are you sure this isn't breaking non-ESP8266 devices? Wouldn't a fix like this be a tad safer: https://github.com/sandeepmistry/arduino-LoRa/pull/257/files